### ISSUE TITLE
using both key settings together doesn't work properly

### ISSUE THREAD

**Issue by sg29176** [Drewol/unnamed-sdvx-clone#477]
**To Reproduce**
1. Set primary/alternate key.
    - my settings
        primary BT: S,D,K,L
        alternate BT: H,J,F,G
2. Try to hit the jack or double trill by using both key settings.
    Ex:
    - BT-0 Jack with S,H,S,H,.....
    - BT-0,1 & 1,2 Trill with (S,D),(J,K),(S,D),(J,K),......
3. When primary key is holded,  alternate key hit couldn't work properly.

**Expected behavior**
When alternate BT was pressed while primal key is holded, operates alternate key hit as new BT hit.

**Comment by 123jimin** [comment]
Hitting the same patterns with a controller also requires not hitting a BT button when it's already pressed.

While the current behavior makes playing with a keyboard quite harder compared to a controller, the alternative behavior would give a huge, unfair advantage to keyboard players compared to controller players, as those patterns would become something trivial.

Moreover, SDVX konasute, and IIRC k-shoot mania, behave identically to USC, so I oppose changing the current behavior.

**Comment by inv41idu53rn4m3** [comment]
I agree with @123jimin but there's still something that needs to be fixed.

If you press primary, then secondary, then release primary, the secondary press gets ignored. This means you can't start a hold note with one hand and complete it with the other like you can on controller.
